packages/semantic-ui/test/Form.test.js

```diff
@@ -4,6 +4,12 @@ import renderer from "react-test-renderer";
 
 import Form from "../src/index";
 
+/** Mock the `react-component-ref` component used by semantic-ui to simply render the children, otherwise tests fail */
+jest.mock("@fluentui/react-component-ref", () => ({
```

Comment:
Should this be semantic-ui

Reply:
@jacqueswho Actually, it needs to be `@fluentui/react-component-ref`... Looking at the [semantic-ui-react source code](https://github.com/Semantic-Org/Semantic-UI-React/blob/master/src/elements/Button/Button.js#L1) you can see that is what needs to be mocked